packages/babel-core/src/transformation/file/options/config.js

```diff
@@ -183,5 +183,11 @@ module.exports = {
   moduleId: {
     description: "specify a custom name for module ids",
     type: "string"
-  }
+  },
+
+  passPerPresset: {
+    description: "Whether to spawn a traversal pass per a preset. By default all presets are merged.",
+    type: "boolean",
+    default: false
```

Comment:
you can add `hidden: true` so it doesn't show up on the website I think. cc @thejameskyle